### PR TITLE
Bump [v115] Upgrades appservices to latest

### PR DESCRIPTION
**Pull request by tarikeshaq**
No ticket bump to latest app services nightly

Changes the FML script so it falls back to the taskcluster index if it doesn't find the FML in the github release

cc @bendk @isabelrios

### CODE REVIEW

File: bin/nimbus-fml.sh [hunk 1]
```diff
@@ -164,14 +164,22 @@ if [ -n "$MOZ_APPSERVICES_LOCAL" ] ; then
     LOCAL_FML_DIR="$MOZ_APPSERVICES_LOCAL/components/support/nimbus-fml"
     export BINARY_PATH="$HOME/.cargo/bin/cargo run --manifest-path $LOCAL_FML_DIR/Cargo.toml --"
 else
-    # Otherwise, we should download a pre-built copy from github.com/mozilla/application-services/releases
+    # Otherwise, we should download a pre-built copy
+    AS_DOWNLOAD_URL="https://github.com/mozilla/application-services/releases/download/v$AS_VERSION"
```
Comment (tarikeshaq) [inline bin/nimbus-fml.sh:168]: @bendk We should file a ticket to remember to switch this if we move the artifacts out of github releases for the 115 release

**Comment by isabelrios** [comment]
@tarikeshaq @bendk The build is still failing, this is the error we see now when building the app:
```
❌  /Users/vagrant/git/Tests/SyncTests/MetaGlobalTests.swift:90:71: cannot specialize non-generic type 'Record'
        let record = Record(id: "keys", payload: keys.asPayload()) as Record<CleartextPayloadJSON>
        ^
```
Bitrise latest logs: https://app.bitrise.io/build/dfd92c01-6809-461b-84dd-088224567bee

**Comment by bendk** [comment]
Turtles all the way down!  I believe this is caused by a name conflict with the remote settings client that was recently added.  I think we should be able to figure out something today.

Sorry that this is dragging out, but at least now we seem to be revealing real issues that would have needed to be fixed at some point.

**Comment by isabelrios** [comment]
> Turtles all the way down! I believe this is caused by a name conflict with the remote settings client that was recently added. I think we should be able to figure out something today.
> 
> Sorry that this is dragging out, but at least now we seem to be revealing real issues that would have needed to be fixed at some point.

Sure!no worries, this is the point of creating this automation, so that we can detect issues as soon as possible in the process. It is usually easier to fix and deploy a new version.. 
Let us know when you have a fix if you want us to try it out.

cc @jjSDET

**Comment by tarikeshaq** [comment]
This should be good to land (bit rise is green!)

**Review by isabelrios — APPROVED** [review]

**Comment by isabelrios** [comment]
@tarikeshaq, @bendk  wondering if we would need same fix on Focus or when the new AS version is taken there, it would work...

**Pull request merged**